android/app/src/main/java/org/fossasia/openevent/fragments/SpeakersListFragment.java

```diff
@@ -118,6 +120,29 @@ public void onRefresh() {
                     public void accept(@NonNull List<Speaker> speakers) throws Exception {
                         mSpeakers.clear();
                         mSpeakers.addAll(speakers);
+                        List<Speaker> sorted_speakerCountry = dbSingleton.getSpeakerList("country");
+                        List<Speaker> sorted_speaker_org = dbSingleton.getSpeakerList("organisation");
```

Comment:
I think using using RxJava to compute countries and organisations off the main thread will be fine